sxg_rs/src/lib.rs

```diff
@@ -82,16 +88,18 @@ impl SxgWorker {
         // https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#section-3.5-7.9.1
         let (mice_digest, payload_body) = crate::mice::calculate(payload_body, 16384);
         let signed_headers = payload_headers.get_signed_headers_bytes(status_code, &mice_digest);
+        let cert_url = self.create_url(&self.config.cert_url_dirname, &self.cert_basename());
+        let validity_url = self.create_url(&self.config.validity_url_dirname, "validity");
         let signature = signature::Signature::new(signature::SignatureParams {
-            cert_url: &self.config.cert_url,
-            cert_sha256: utils::get_sha(&self.config.cert_der),
+            cert_url: &cert_url,
+            cert_sha256: self.config.cert_sha256.clone(),
```

Comment:
Declare `SignatureParams::cert_sha256` as type `&'a [u8]`, and this cloning could be removed.